README.md

```diff
@@ -1,13 +1,30 @@
 # Project Auth
 
-Replace this readme with your own information about your project.
+Pair-programming project with Nazanin Adampira🙋‍♀️
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+A sign in/up form that uses authenticate the users passwords. Once signedup/signed in the user get access to some well-being questions.
 
-## The problem
+## The build 🛠
+We built an API with authentication to implement a registration flow, and a frontend with forms to register, sign in, and view some content once users logged in. 
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Our API includes:
+- Registration endpoint, to create a new user.
+- Sign-in endpoint, to authenticate a returning user.
+- An authenticated endpoint which only returns content if the `Authorization` header with the user's token was correct.
 
-## View it live
+Our frontend has
+- A sign-up/sign-in form.
+- A page (in Redux) to show the authenticated content from the API.
+- A 'sign out' button that removes the saved access token and redirects the user to the login form.
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+### Tech👩‍💻
+
+- MongoDB
+- Node.js
+- Express
+- Redux
+- React
+
+#### View it live 👀
+Frontend: https://infallible-engelbart-8dce94.netlify.app/
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```

![image](https://user-images.githubusercontent.com/50926159/120885466-5c359780-c5e9-11eb-9a74-92fb204e40a7.png)

Reply:
hmmm strange, I have sent the netlify link to two people and it works for them? Can you try again and see if the problem is still there😅? thank you ! https://infallible-engelbart-8dce94.netlify.app/

Reply:
It is what happened when you refresh your page.